.gitignore

```diff
@@ -1,4 +1,5 @@
 node_modules/
```

Comment:
Please delete the package-lock.json file  We already have a yarn.lock which is what we'll use for package management. Check out Yarn if you haven't yet, it's great!

Reply:
Done.